Resources/Prototypes/Voice/speech_emotes.yml

```
@@ -20,11 +20,17 @@
     - yells
     - yelled
     - yelling
+    - крик
+    - кричит
+    - ор
+    - орёт
+    - визг
+    - визжит
```

Отметь изменения блоками
```suggestion
    # Corvax-Localization-Start
    - крик
    - кричит
    - ор
    - орёт
    - визг
    - визжит
    # Corvax-Localization-End
```